EIP-1480.md

```diff
@@ -191,13 +191,13 @@ There are no backwards compatibility concerns.
 
 <!--Test cases for an implementation are mandatory for EIPs that are affecting consensus changes. Other EIPs can choose to include links to test cases if applicable.-->
 
-DAOstack ERC-TBD implementation includes test cases written using Truffle.
+DAOstack ERC-1480 implementation includes test cases written using Truffle.
 
 ## Implementation
 
 <!--The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.-->
 
-DAOStack full implementation is available [here](ADD LINK)
+DAOStack full implementation is available [here](https://github.com/daostack/access_control)
```

Comment:
`DAOstack` not `DAOStack`.
Should also have a period at the end of the sentence.